arch/x86/include/asm/barrier.h

```diff
@@ -87,9 +127,37 @@ do {									\
 
 #endif
 
+#else /* CONFIG_KTSAN */
+
+#define smp_store_release(p, v)						\
+do {									\
+	typeof(p) ___p1 = (p);						\
+	compiletime_assert_atomic_type(*___p1);				\
+	ktsan_sync_release((void *)___p1);				\
+	asm volatile("mfence":::"memory");				\
+	ACCESS_ONCE(*___p1) = (v);					\
+} while (0)
+
+#define smp_load_acquire(p)						\
+({									\
+	typeof(p) ___p1 = (p);						\
```

Comment:
add a TODO that this must be replaced with ktsan_atomic_load(p, ktsan_memory_order_acquire)